src/app/help/en/welcome.md

```diff
@@ -190,24 +189,24 @@ The more Steem Power you have in your account, the more your upvotes will be wor
 
 Payouts are made 7 days after the post/comment is created. You can claim your rewards in your wallet after 7 days.
 
-The payments may fluctuate (up and down) until the final payment is reached.
+The projected payments may fluctuate (up and down) until the final payment amount is reached.
 
 Payments for posts are split between the author (at least 75%) and the curators (up to 25%).
 
-The author reward is paid 50% in Steem Power, and 50% in liquid STEEM/SBD.
+The author reward is paid 50% in Steem Power and 50% in STEEM/SBD by default.
 
 Authors also have the option to decline payout, or be paid in 100% Steem Power!
 
 <a href="#Table_of_Contents">^</a>
 ### <span id="Reputation">Reputation</span>
 
-A reputation score is one way Steemit measures the amount of value you have brought to the community.
+A reputation score is one way that Steemit measures the amount of value you have brought to the community.
 
-The higher the number, the more weighted votes an account has earned.
+The higher the reputation, the more upvotes the account received.
 
 All new users start at 25.
 
-Your reputation will go up as you earn upvotes for your posts and comments, but it can come down if they are flagged.
+Your reputation will go up as you earn upvotes for your posts and comments, but it can come down if they are downvoted.
```

Comment:
The UI refers to downvotes as flags.

Reply:
The word 'flag' was confusing to users; particularly to those who were translating.  The thought process behind changing it to 'downvoted' was to clarify it as being opposite of upvoting, which is mentioned in the same sentence.  It is meant to make the sentence easier to understand for users who are not yet familiar of how upvotes and flags look like on a post.

Reply:
Still it's not consistent with current UI, so IMO it should be changed *only if* flag will be renamed to downvote.

Reply:
A happy medium may be to go with the verbiage used above:
_downvote (flag)_

Reply:
Depending on the language to which it is being translated into, there is no way possible to literally translate some terms. The word flag may in this case be used as a term for downvoting however when translated into some other language it won't make any sense as many a language group already have their terms for many such actions.

The one that always comes to mind is the term "Like" on one of the other well known social networks out there. In many a country located in the Balkans, just as one example that I am fully familiar with, they use the term "lajkanje" which can't be found in any dictionary, yet it is understood by one and all what it means. Hence I would have to agree with a logical approach which in this case would be as TimCliff suggested "downvote (flag)".

Same applies to the word Curation, the literal translation of this word would make absolutely no sense in the language used in some of the countries of the Balkan region. Hence in the translation it would have to use wording that describes the action with the English word in brackets. So as to ensure that there is nothing lost in translation and that the original English version takes precedence in such situations.

This is not a project that any "official translator" or "online automated translator" can accomplish for any one language. 

To back this statement up, my wife who's first language is Croatian and English is her second language who has gone ahead and taken the initiative to do such a translation of the front page came across this situation on numerous points and when I compared her correct translation as per the grammar of that language to what the "online translator" gave me, it was way different. Adding to the need for a logical approach to translating pages, and not some automated or official literal translation which shall not make any sense to people reading it. 

ref: https://steemit.com/steemit/@sylviamiller/translation-of-the-welcome-page-croatian-hrvatski-finished

anyone is more than welcome to go and use an online translator and compare the differences. 

Jack

Reply:
That was one of the main points behind getting it optimized and to pave the way for translation.